pkg/kubelet/pod_container_deletor.go

```diff
@@ -35,17 +34,17 @@ type containerStatusbyCreatedList []*kubecontainer.ContainerStatus
 
 type podContainerDeletor struct {
 	containersToKeep int
-	runtime          internalapi.RuntimeService
+	runtime kubecontainer.Runtime
 }
 
 func (a containerStatusbyCreatedList) Len() int           { return len(a) }
 func (a containerStatusbyCreatedList) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 func (a containerStatusbyCreatedList) Less(i, j int) bool { return a[i].CreatedAt.After(a[j].CreatedAt) }
 
-func newPodContainerDeletor(runtime kubecontainer.Runtime, containersToKeep int) *podContainerDeletor {
+func newPodContainerDeletor(_runtime kubecontainer.Runtime, containersToKeep int) *podContainerDeletor {
```

Comment:
same here? why the leading underscore?

Reply:
Just changed to kubeRuntime for the function paraemeter name

Reply:
committed fixes.